package.json

```diff
@@ -16,25 +16,30 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@babel/core": "7.0.0",
```

Comment:
Should these all be the latest version, 7.4.4?  https://github.com/babel/babel/releases

We should also avoid relative deps (`^` or `~`) since it makes for inconsistent `npm install` for consuming apps.  To be safe, we make upgrades explicit